tests/console/zypper_lifecycle.pm

```diff
@@ -41,6 +41,11 @@ sub lifecycle_output_check {
         record_info 'poo#129026';
         return;
     }
+    # https://suse.slack.com/archives/C02D16TCP99/p1688366603014179
+    if (is_sle('=15-sp3') && $output =~ /Containers Module.*2023-06-30/) {
+        record_soft_failure 'https://jira.suse.com/servicedesk/customer/portal/1/MSC-658';
```

Comment:
```
* Please add a bug/ticket reference for `record_soft_failure`, otherwise
  CI checks may fail. you can use formats like below:
  bsc#12345 -> Bugzilla bug
  poo#12345 -> Progress ticket
  jsc#SLE-19640 -> Jira ticket
  Maniphest#T5531
  fate.suse.com/123
  $reference -> if you have to use a variable to define a reference ticket
  If you don't have a reference ticket, and still want to mark a specific
  step as soft_fail, please use `record_info` with softfail tag:
  record_info($title [, $output] [, result => softfail] [, resultname => $resultname]);
```

Suggest change:

record_info('Softfail', "https://jira.suse.com/servicedesk/customer/portal/1/MSC-658", result => 'softfail');

Reply:
or
``` 
my $reference = "https://jira.suse.com/servicedesk/customer/portal/1/MSC-658";
record_soft_failure "$reference";
```